bazel/workspace_rule.bzl

```diff
@@ -39,22 +40,43 @@ def _build_http_archive(
 
   # Generate http_archive rule
   if build_file:
-    http_archive(
-      name = name,
-      urls = urls,
-      strip_prefix = prefix,
-      build_file = build_file,
-      patches = patches,
-      patch_args = patch_args,
-    )
+    if sha256:
```

Comment:
So these nested if statements suck, are there any language features I can use to avoid this?

Reply:
It should be okay to pass "None" to `sha256`. That's already the default if the parameter is unset in `http_archive`.

I think you can revert the nesting that you added an it should work.

Reply:
Bueno. Done